man/journal-upload.conf.xml

```diff
@@ -74,6 +74,15 @@
         <listitem><para>SSL CA certificate.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>NetworkTimeoutSec=</varname></term>
+
+        <listitem><para>When network connectivity to the server is lost, this option
+        configures the time to wait for the connectivity to get restored. If the server is
+        not reachable over the network for the configured time, systemd-journal-upload exits.
```

Comment:
please enclose systemd-journal-upload in `<command></command>`.

And please add `The specified time is in seconds, except if a different time unit is explicitly specified. For details see <citerefentry><refentrytitle>systemd.timer</refentrytitle><manvolnum>5</manvolnum></citerefentry>.`

or so, to explain the unit situation

Reply:
Due to reasons mentioned at end-of-page-comment, these changes are in this new pull request: https://github.com/systemd/systemd/pull/19827